docs/src/modules/components/ApiPage/list/ClassesList.tsx

```diff
@@ -25,12 +29,26 @@ const StyledApiItem = styled(ExpandableApiItem)(
     '& .prop-list-class': {
       margin: 0,
     },
+    '&.classes-list-deprecated-item': {
+      '& .MuiApi-item-note': {
+        color: `var(--muidocs-palette-warning-700, ${lightTheme.palette.warning[700]})`,
+      },
+    },
+    '& .classes-list-alert': {
+      marginTop: 12,
+      marginBottom: 16,
+    },
   }),
   ({ theme }) => ({
     [`:where(${theme.vars ? '[data-mui-color-scheme="dark"]' : '.mode-dark'}) &`]: {
       '& .prop-list-title': {
         color: `var(--muidocs-palette-grey-50, ${darkTheme.palette.grey[50]})`,
       },
+      '&.classes-list-deprecated-item': {
+        '& .MuiApi-item-note': {
+          color: `var(--muidocs-palette-warning-400, ${lightTheme.palette.warning[400]})`,
```

Comment:
Is there a specific reason to use the `lightTheme` instead of dark one ?

Reply:
It shouldn't be 😅 I fixed it 👍🏼